public/components/custom_panels/home.tsx

```diff
@@ -153,59 +163,85 @@ export const Home = ({ http, chrome, parentBreadcrumb }: Props) => {
             },
           ];
         });
-        setToast(`Custom Panel "${clonedCustomPanelName}" successfully created!`);
+        setToast(`Operational Panel "${clonedCustomPanelName}" successfully created!`);
         return res.body.id;
       })
       .catch((err) => {
         setToast(
-          'Error cloning Custom Panel, please make sure you have the correct permission.',
+          'Error cloning Operational Panel, please make sure you have the correct permission.',
           'danger'
         );
         console.error(err.body.message);
       });
   };
 
-  // Deletes an existing Custom Panel
+  // Deletes an existing Operational Panel
   const deleteCustomPanel = (customPanelId: string, customPanelName?: string, showToast = true) => {
     return http
       .delete(`${CUSTOM_PANELS_API_PREFIX}/panel/` + customPanelId)
       .then((res) => {
         setcustomPanelData((prevCustomPanelData) => {
           return prevCustomPanelData.filter((customPanel) => customPanel.id !== customPanelId);
         });
-        if (showToast) setToast(`Custom Panel "${customPanelName}" successfully deleted!`);
+        if (showToast) setToast(`Operational Panel "${customPanelName}" successfully deleted!`);
         return res;
       })
       .catch((err) => {
         setToast(
-          'Error deleting Custom Panel, please make sure you have the correct permission.',
+          'Error deleting Operational Panel, please make sure you have the correct permission.',
           'danger'
         );
         console.error(err.body.message);
       });
   };
 
   return (
-    <>
+    <div>
       <EuiGlobalToastList
         toasts={toasts}
         dismissToast={(removedToast) => {
           setToasts(toasts.filter((toast) => toast.id !== removedToast.id));
         }}
         toastLifeTimeMs={6000}
       />
-      <CustomPanelTable
-        loading={loading}
-        fetchCustomPanels={fetchCustomPanels}
-        customPanels={customPanelData}
-        createCustomPanel={createCustomPanel}
-        setBreadcrumbs={chrome.setBreadcrumbs}
-        parentBreadcrumb={parentBreadcrumb}
-        renameCustomPanel={renameCustomPanel}
-        cloneCustomPanel={cloneCustomPanel}
-        deleteCustomPanel={deleteCustomPanel}
-        setToast={setToast}
+      <Route
+        exact
+        path={renderProps.match.path}
+        render={(props) => {
+          return renderPageWithSidebar(
+            <CustomPanelTable
+              loading={loading}
+              fetchCustomPanels={fetchCustomPanels}
+              customPanels={customPanelData}
+              createCustomPanel={createCustomPanel}
+              setBreadcrumbs={chrome.setBreadcrumbs}
+              parentBreadcrumb={parentBreadcrumb}
+              renameCustomPanel={renameCustomPanel}
+              cloneCustomPanel={cloneCustomPanel}
+              deleteCustomPanel={deleteCustomPanel}
+              setToast={setToast}
+            />,
+            4
```

Comment:
i removed the requirement to put index, could you pull latest commits and try removing the second argument

Reply:
Will rebase and then make this change.